README.md

```diff
@@ -19,13 +19,15 @@ pipenv run ./scripts/run_app.sh
 
 ## Testing
 
-To test the app:
+To run the app's unit tests:
 
 ```
-pipenv run ./scripts/run_app.sh
+pipenv run ./scripts/run_tests_unit.sh
 ```
 
-then, in another terminal window/tab, navigate to a checked out copy of eq-survey-runner:
+To test the apps functionality:
+
+In another terminal window/tab, navigate to a checked out copy of eq-survey-runner:
```

Comment:
This no longer makes sense?
Why have we removed the instruction on how to run the app? should we not have added this as well as?

Reply:
I figured the sensible person would know that to test the apps functionality, you need to have the app running.
But fair enough, I like when READMEs are nice and explicit so I'll add it again

Reply:
Apologies, i didn't realise that above it detailed how to run the app.
In the small diff i saw on here it looked as though the instructions had been removed.